plugins/registry.js

```diff
@@ -248,6 +281,35 @@ export default class PluginRegistry {
         return id;
     }
 
+    // Register a post sub menu list item by providing some text and an action function.
+    // Accepts the following:
+    // - text - A string or React element to display in the menu
+    // - action - A function to trigger when component is clicked on
+    // - filter - A function whether to apply the plugin into the post' dropdown menu
+    // Returns a unique identifier.
+    registerPostDropdownMenuItem(text, action, filter) {
```

Comment:
Should we rename this to something that indicates that it is registering a submenu specifically, it's a bit confusing with the existing `registerPostDropdownMenuComponent`? Maybe `registerPostDropdownSubMenuItem`?

Reply:
Great idea.  Done.